### PR TITLE
Add swap_blocks feature to improve accuracy

### PR DESCRIPTION
**Pull request by catid-saronic**
The design of your model didn't match my intuition in that the blocks grow in width 2x each time.  It made more sense to me if the middle block was wider and gradually converged on the two outputs.  When parameter matched this means the sizes are (1x, 6x, 2x) rather than (1x, 2x, 4x).  This ends up improving test set accuracy of the model by about +3%.

I only modified the training script and simplified the setup so all the scripts after that would need to be updated.  For this reason I don't recommend you merge this change but just providing the diff as reference.

### CODE REVIEW

**Comment by LorenzoLamberti94** [comment]
hi, thanks for the comment. If you can comment on the main page, it helps me in keeping track of pull requests

https://github.com/pulp-platform/pulp-dronet
illl take a look
